### DISCUSSION TITLE
How to find clients?

### DISCUSSION THREAD

**Discussion by mstanciu552** [EddieHubCommunity/support#534] [Q&A]
Can you share your expirience on getting clients when you are new to freelancing?
Thank you in advance.

**Comment by eddiejaoude** [comment]
Sorry for my slow reply @mstanciu552 

There is some overlap with this discussion https://github.com/EddieJaoudeCommunity/support/discussions/261

**networking**
- There are platforms for freelancers, I have not used them to find work, but I have hired people from there. I hear they are a race to the bottom on pricing, but I do not know. Might be a place to start though and you can break away later
- Contact and connect with agencies who have a lot of work on, they can share their work load - you will make less money with this, but helps to grow your network, confidence and some savings

**portfolio**
- have example projects and ideally makes these open source as their techies will want to have a look
- 

**tips**
- Under promise and over delivery, that way people will sing your praises to their, network, friends and colleagues
- get started, the beginning will be hard, but the sooner you start the better


I hope that helps 🤓
